source/components/handling-events.md

```diff
@@ -0,0 +1,129 @@
+Instead of having to register event listeners on elements you'd like to
+respond to, simply implement the name of the event you want to respond to
+as a method on your component.
+
+For example, imagine we have a template like this:
+
+```hbs
+{{#x-clickable}}
+  This is a clickable area!
+{{/x-clickable}}
+```
+
+Let's implement `x-clickable` such that when it is
+clicked, an alert is displayed:
+
+```app/components/x-clickable.js
+export default Ember.Component.extend({
+  click: function(evt) {
+    alert("XClickableComponent was clicked!");
+  }
+});
+```
+
+Browser events may bubble up the DOM which potentially target parent component(s)
+in succession. To enable bubbling `return true;` from the event handler method
+in your component.
+
+```app/components/x-clickable.js
+export default Ember.Component.extend({
+  click: function(evt) {
+    Ember.Logger.info("XClickableComponent was clicked!");
+    return true;
+  }
+});
+```
+
+Any registered event can be defined as an event handler in your component.
+(See The list of event names at the end of this page.)
+
+## Sending Actions
+
+In some cases your component needs to define event handlers, perhaps to support
+various draggable behaviors. For example, a component may need to send an `id`
+when it receives a drop event:
+
+```hbs
+{{x-draggable action="didDrop"}}
+  {{! …an element as a drop affordance may go in here… }}
```

Comment:
a ui term for an element that you can interact with, I'll remove it, it distracts from the topic